include/language-deprecation.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Holds everything related to deprecated properties of `PLL_Language`.
+ *
+ * @since 3.4
+ */
+abstract class PLL_Language_Deprecation {
+
+	/**
+	 * List of deprecated term properties and related arguments to use with `get_tax_prop()`.
+	 *
+	 * @private
+	 *
+	 * @var array[]
```

Comment:
```suggestion
	 * @var string[][]
```

Reply:
Done in [ecfb23b](https://github.com/polylang/polylang/pull/1227/commits/ecfb23bdf84acf8ee8c182474c1dd69c7137028e).